presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2656,6 +2656,9 @@ public void testNonEqualityJoin()
         assertQuery(
                 "SELECT COUNT(*) FROM (VALUES 1, 2) t1(a) JOIN (VALUES 10, 11) t2(b) ON a > 2",
                 "VALUES (0)");
+        assertQuery(
```

Comment:
Yeah - it is not supposed to. It is just for coverage.
I aggree the proper plan-shape test is missing. Do we really want it provided it does not exist for any other PP functionality?